api-admin/src/main/resources/api/openapi.yml

```diff
@@ -1707,6 +1707,58 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/dataImports:
+    get:
+        summary: Gets a list of imports for a study
+        tags: [ dataImport ]
+        operationId: get
+        parameters:
+          - { name: portalShortcode, in: path, required: true, schema: { type: string } }
+          - { name: studyShortcode, in: path, required: true, schema: { type: string } }
+          - { name: envName, in: path, required: true, schema: { type: string } }
+        responses:
+          '200':
+            description: import list
+            content: { application/json: { schema: { type: object } } }
+          '500':
+            $ref: '#/components/responses/ServerError'
+
+    delete:
+      summary: delete imports for a study
+      tags: [ dataImport ]
+      operationId: delete
+      parameters:
+        - { name: portalShortcode, in: path, required: true, schema: { type: string } }
+        - { name: studyShortcode, in: path, required: true, schema: { type: string } }
+        - { name: envName, in: path, required: true, schema: { type: string } }
+        - { name: importId, in: query, required: true, schema: { type: integer } }
+      responses:
+        '200':
+          description: import list
+          content: { application/json: { schema: { type: object } } }
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/import/{importFileName}:
```

Comment:
is it intentional that this lives under `/api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/import/` versus the two above it living under `/api/portals/v1/{portalShortcode}/studies/{studyShortcode}/env/{envName}/dataImport/`?